### PR TITLE
Update customer confirmation email

### PR DESCRIPTION
**Pull request by ethatcher83**
update CertificateOrderNotificationModel for new fields
add method to map output for email copy required output
set new mapped values to email object
update text being output for what happens next
add unit tests
update email integration test
add tilt file

Resolves: BI-10972

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/ordernotification/config/ApplicationConfig.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ Map<DeliveryMethodTuple, String> deliveryMethodMappings(@Value("${email.dispatch
         mappings.put(new DeliveryMethodTuple(DeliveryMethodApi.POSTAL, DeliveryTimescaleApi.STANDARD),
                 "Standard delivery (aim to dispatch within "+dispatchDays+" working days)");
         mappings.put(new DeliveryMethodTuple(DeliveryMethodApi.POSTAL, DeliveryTimescaleApi.SAME_DAY),
-                "Same Day");
+                "Express (Orders received before 11am will be dispatched the same day. Orders received after 11am will be dispatched the next working day)");
```
Comment (VaughanJackson123) [inline src/main/java/uk/gov/companieshouse/ordernotification/config/ApplicationConfig.java:74]: Should there be another full stop before the closing bracket?

File: src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertificateOrderNotificationModel.java [hunk 2]
```diff
@@ -175,6 +193,7 @@ public boolean equals(Object o) {
         return Objects.equals(certificateType, that.certificateType) &&
                 Objects.equals(statementOfGoodStanding, that.statementOfGoodStanding) &&
                 Objects.equals(deliveryMethod, that.deliveryMethod) &&
+                Objects.equals(emailCopyRequired, that.emailCopyRequired) &&
```
Comment (VaughanJackson123) [inline src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertificateOrderNotificationModel.java:197]: Has `deliveryTimescale` been omitted intentionally?

**Review by VaughanJackson123 — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertificateOrderNotificationModel.java (hunk 2 above)
Comment (ethatcher83) [reply src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertificateOrderNotificationModel.java:197]: Should be there, will update

File: src/main/java/uk/gov/companieshouse/ordernotification/config/ApplicationConfig.java (hunk 1 above)
Comment (ethatcher83) [reply src/main/java/uk/gov/companieshouse/ordernotification/config/ApplicationConfig.java:74]: Checking the ticket and it does not specify a full stop

**Pull request merged**